client/storybook/main.js

```diff
@@ -5,7 +5,7 @@ module.exports = {
     {
       directory: '../../wagtail/admin/templates/wagtailadmin/shared/',
       titlePrefix: 'Shared',
-      files: '*.stories.*',
+      files: '**/*.stories.*',
```

Comment:
This is so all stories within this at all levels show up within this "Shared" section.